test/Parse/enum.swift

```diff
@@ -552,3 +552,10 @@ enum SE0155 {
   // expected-note@-1 {{did you mean to remove the empty associated value list?}} {{17-18=}}
   // expected-note@-2 {{did you mean to explicitly add a 'Void' associated value?}} {{17-17=Void}}
 }
+
+
+// SR-11261
+enum SR11261 {
+  case identifier
+  case operator // expected-error {{keyword 'operator' cannot be used as an identifier here}} expected-error {{expected pattern}} expected-note {{if this name is unavoidable, use backticks to escape it}} 
```

Comment:
Just to cover more bases,

- Can you add a `case` declaration after this one for a valid enum case to make sure the munging works.
- Can you add a multi-case declaration that uses an operator

(For the latter, you don’t need to improve the diagnostics in this patch if it goes haywire, it’s just good to have what it does down!)

Reply:
Added @CodaFi 👍